.changeset/afraid-coats-rescue.md

```diff
@@ -0,0 +1,9 @@
+---
+"@remix-run/web-blob": patch
+"@remix-run/web-fetch": patch
+"@remix-run/web-file": patch
+"@remix-run/web-form-data": patch
+"@remix-run/web-stream": patch
+---
```

Comment:
Should this be a minor since it's sort of adding a new feature in CJS files?

Reply:
I went with `patch` as it not a _real_ feature, but I can go with `minor` if you want

Reply:
ok - yeah I think it could go either way since there's no userland _code_ API changes.  Going to update to `minor` just to be safe since we're providing a new surface area for the bundler